### ISSUE TITLE
[Request] 4th main section: build a Polymer component library

### ISSUE THREAD

**Issue by kevinSuttle** [Polymer/docs#1780]
https://www.polymer-project.org/1.0/start/

Right now, the 3 are:

1. Use and element
2. Build an element
3. Build an app

**Comment by robdodson** [comment]
@arthurevans @tjsavage this could be a cool thing to work on after the summit. There are a lot of large orgs pursuing this and it presents unique challenges like running tests across all elements, how to properly version a catalog of elements, etc.

**Comment by tjsavage** [comment]
Agreed - we should definitely look into building up a package of content around how to scale Polymer & Web Component deployments at a company. The Summit talks will provide a huge start for this.

**Comment by kevinSuttle** [comment]
:+1: Let me know if you need some help.
